sa/sa.go

```diff
@@ -2299,3 +2300,17 @@ func (ssa *SQLStorageAuthority) GetRevokedCerts(req *sapb.GetRevokedCertsRequest
 
 	return nil
 }
+
+func (ssa *SQLStorageAuthority) GetLastExpiration(ctx context.Context, req *emptypb.Empty) (*timestamppb.Timestamp, error) {
```

Comment:
Totally optional, but this name came to me while I was reading the code.

```suggestion
func (ssa *SQLStorageAuthority) GetMaxExpiration(ctx context.Context, req *emptypb.Empty) (*timestamppb.Timestamp, error) {
```

Reply:
Good call, done